### PR TITLE
SD-JWT Kotlin Library Project Proposal

### PR DESCRIPTION
**Pull request by fabian-hk**
This library implements the [Selective Disclosure for JWTs](https://www.ietf.org/archive/id/draft-ietf-oauth-selective-disclosure-jwt-04.html) specification, which is an important building block for future digital identity wallets. Since it is written in Kotlin, it can be used widely, e.g. in web backends or on Android.

### CODE REVIEW

File: projects/sd-jwt-kotlin.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+This library implements the Selective Disclosure for JWTs specification, which is an important building block for future digital identity wallets. Since it is written in Kotlin, it can be used widely, e.g. in web backends or on Android.
+
+# Current Code of Conduct
+
+--
+
+# TAC Sponsor
+
+--
```
Comment (tkuhrt) [inline projects/sd-jwt-kotlin.md:19]:
```suggestion
- [Tracy Kuhrt](https://github.com/tkuhrt)
```

**Review by tkuhrt — APPROVED** [review]
Thank you for being the first to propose a project. This will be a great project to bring into the OpenWallet Foundation. I am happy to serve as a sponsor for this project although for a lab, this is not required.

A couple of points:
1. Before we bring in the source base to the OWF, one of the requirements is that the code has [DCO sign off](https://developercertificate.org/). Please let me know if you need instructions on doing this.
2. What are your thoughts about making this a generic SD-JWT lab and bringing in a Python implementation as well?

**Review by skounis — APPROVED** [review]

**Comment by fabian-hk** [comment]
Thank you for your response.
1. Yes I need help how to integrate the DCO sign off into my repository. Do I just add the file to the repo?
2. I am not sure if it is good to make a general proposal for SD-JWT libraries, because they will probably not develop at the same speed. This will be strongly influenced by the popularity of the programming language and the number of maintainers. So I personally think it is better to keep the proposals separate, unless it is no problem if the libraries do not always have the same feature level.

**Comment by tkuhrt** [comment]
> 1. Yes I need help how to integrate the DCO sign off into my repository. Do I just add the file to the repo?
Here is a [guide for bulk fixing commits without losing history](https://hyperledger-indy.readthedocs.io/projects/sdk/en/latest/docs/contributors/signing-commits.html)

> 2. I am not sure if it is good to make a general proposal for SD-JWT libraries, because they will probably not develop at the same speed. This will be strongly influenced by the popularity of the programming language and the number of maintainers. So I personally think it is better to keep the proposals separate, unless it is no problem if the libraries do not always have the same feature level.
In other open projects that I am involved in, the community that is working on similar items likes to work together as a single “project” even though the code advances at different rates. The benefit of doing something like this is to allow for learning. If you choose to go this direction, we will still create separate repos for each of the language implementations and they would be managed separately with different (perhaps overlapping) maintainers as determined by each repo. The advancement of these repos may end up going at different speeds, but would allow the people working on SD-JWT within the OpenWallet Foundation to be part of a larger community.

**Comment by tkuhrt** [comment]
Hi, @fabian-hk. We will be reviewing this pull request in this week's TAC meeting on Wednesday May 17 at 7:00 AM US/Pacific. It would be great if you can join us. Please see the [OpenWallet Foundation calendar](https://calendar.google.com/calendar/u/0?cid=Y181MjdhMWRiNjIxZmU3MDczMzg0MjFhOWM0YzdlZjJmNjE2MGI5NzM4MzhjZmRiYjUwZjBkYjMyYWQwZmQyMGViQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20) for details on the meeting.

**Comment by fabian-hk** [comment]
Hi, @tkuhrt  I will attend the meeting today and I have followed your guide on DCO signing my commits. Can you check if it is correct now?

**Comment by fabian-hk** [comment]
I have renamed the ``master`` branch to ``main`` and I have transferred the repo to you. Can you check if it worked?

**Comment by tkuhrt** [comment]
I also created the #sd-jwt-kotlin channel on the [OpenWallet Foundation Discord](https://discord.com/invite/yjvGPd5FCU).

**Pull request merged**